main.py

```diff
@@ -1,7 +1,57 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Actions",
+        "Dramma",
+    ]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        first_name, last_name = actor
+        Actor.objects.create(first_name=first_name, last_name=last_name)
```

Comment:
You can unpack values in for loop statement , like `for actor_name , actor_surname in actors:`